lib/maintainers/runner.rb

```diff
@@ -271,7 +289,9 @@ def report(options)
       when 'repo'
         report_repo_details(maintainers_files)
       when 'people'
-        report_people_details(maintainers_files)
```

Comment:
This is not the first time git has highlighted a non-existent diff (and I've never really dug into root causes for it)